src/rpc/handlers/BookChanges.cpp

```diff
@@ -0,0 +1,250 @@
+#include <ripple/app/ledger/Ledger.h>
+#include <ripple/basics/ToString.h>
+
+#include <backend/BackendInterface.h>
+#include <rpc/RPCHelpers.h>
+
+#include <boost/json.hpp>
+#include <algorithm>
+
+namespace json = boost::json;
+using namespace ripple;
+
+namespace RPC {
+
+struct BookChange
+{
+    STAmount sideAVolume;
+    STAmount sideBVolume;
+    STAmount highRate;
+    STAmount lowRate;
+    STAmount openRate;
+    STAmount closeRate;
+};
+
+class BookChangesHandler
+{
+    std::reference_wrapper<Context const> context_;
+    std::map<std::string, BookChange> tally_;
```

Comment:
These aren't initialized until we call reset. 

can we just initialize these here with `= {}`. I don't think it will change anything, I just prefer explicitly instantiating member variables.

Reply:
Why would that be?
Just because i'm not explicitly setting a value in initialization list nor in the members section does not mean default constructor will not be called for a user defined type.
See https://godbolt.org/z/ovnW6erx3

Reply:
Yea, these are exactly the same functionally. 

I prefer explicitly calling the default constructor w/ `= {}` or `Foo foo{}`, but that's just me.

Reply:
Up to you here.